cypress/e2e/with-users/controllers/list.spec.ts

```diff
@@ -1,4 +1,4 @@
-import { generateMAASURL } from "../../utils";
+import { generateId, generateMAASURL } from "../../utils";
```

Comment:
I used this opportunity to fix the failing controller test now that the snap has updated